agent/php_memcached.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "php_memcached.h"
+#include "nr_datastore_instance.h"
+#include "php_agent.h"
+
+nr_datastore_instance_t* nr_php_memcached_create_datastore_instance(
```

Comment:
This looks like a good candidate for unit tests. At least test for invalid input.

Reply:
added in [9ba4554](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/9ba4554faa8946ae0a2258192b48a9e388ec0f67)